shared/getting-started/getDeviceOnDash/intel-nuc.md

```diff
@@ -6,10 +6,12 @@ __Warning:__ {{ $names.os.upper }} will completely overwrite the internal media
 
 Now connect up the power supply and turn the device on. The power button may be a small round button on the top of the device or a square button on the front.
 
-Press the F10 key while the **BIOS** is loading in order to enter the boot menu. Next, select the `UEFI : USB` option from the boot menu so that the device will boot from your USB drive.
+Press the F10 key while the **BIOS** is loading in order to enter the boot menu. Next, select the `UEFI : USB` (or `resinOS`) option from the boot menu so that the device will boot from your USB drive.
```

Comment:
@kb2ma is it still being called resinOS on boot menu?

Reply:
If yes, then just as a side task. Ask the balenaOS team if they are working towards resolving that and on track to changing it to balenaOS

Reply:
Seeing is believing. I will followup with an issue in balena-os/balena-intel repo.

![boot-menu](https://user-images.githubusercontent.com/2944058/114696143-59ee4580-9cea-11eb-9b30-87dbf4bb8735.jpg)

Reply:
Expecting a response on the 'resinOS' name question in balena-os/balena-intel#362.

Reply:
Thanks for creating the issue @kb2ma